axelor-account/src/main/resources/views/InvoiceLine.xml

```diff
@@ -526,6 +527,12 @@
       expr="eval: account?.manageCutOffPeriod ? __parent__?.cutOffEndDate : null"/>
   </action-record>
 
+  <action-record name="action-invoice-line-record-set-tax"
+    model="com.axelor.apps.account.db.InvoiceLine">
+    <field name="taxLine" expr="eval: account?.defaultTax?.activeTaxLine"
+      if="!taxLine &amp;&amp; account &amp;&amp; account.isTaxRequiredOnMoveLine"/>
```

Comment:
why only when the tax is required on MoveLine? for me we should do it all the time if the taxLine is empty and there is a defaultTax on the selected account